WalletWasabi.Fluent/ViewModels/Wallets/MusicBoxCoinjoinProfileSelectorViewModel.cs

```diff
@@ -0,0 +1,96 @@
+using ReactiveUI;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Fluent.ViewModels.CoinJoinProfiles;
+using WalletWasabi.Fluent.ViewModels.Dialogs;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets;
+
+[NavigationMetaData(Title = "Coinjoin Profiles")]
+public partial class MusicBoxCoinjoinProfileSelectorViewModel : RoutableViewModel
+{
+	[AutoNotify] private CoinJoinProfileViewModelBase? _selectedProfile;
+	[AutoNotify] private bool _isFlyoutOpen;
+	[AutoNotify] private bool _showProfile;
+
+	public MusicBoxCoinjoinProfileSelectorViewModel(KeyManager keyManager)
+	{
+		Profiles = DefaultProfiles.ToList();
+		Profiles.Add(new ManualCoinJoinProfileViewModel(keyManager));
+
+		_selectedProfile = IdentifySelectedProfile(keyManager);
+
+		OpenFlyoutCommand = ReactiveCommand.Create(() =>
+		{
+			IsFlyoutOpen = false;
+			IsFlyoutOpen = true;
+		});
+		SelectProfileCommand = ReactiveCommand.CreateFromTask<CoinJoinProfileViewModelBase>(async p => await OnCoinjoinProfileSelectedAsync(keyManager, p));
+
+		this.WhenAnyValue(x => x.SelectedProfile)
+			.Select(x => x is null)
+			.CombineLatest(Services.UiConfig.WhenAnyValue(x => x.PrivacyMode))
+			.Subscribe(x =>
+			{
+				Profiles.ForEach(p => p.IsSelected = p.GetType() == SelectedProfile?.GetType());
+				ShowProfile = !x.First && !x.Second;
```

Comment:
Avoid putting logic inside `Subscribe`.

For example, the `ShowProfile` can be defined like this:

```csharp
var isPrivacyModeActive =  Services.UiConfig.WhenAnyValue(x => x.PrivacyMode);
var selectedProfile = this.WhenAnyValue(x => x.SelectedProfile);

ShowProfile = selectedProfile
    .CombineLatest(isPrivacyModeActive , (selectedProfile, isPrivacyModeActive) => new { selectedProfile, isPrivacyModeActive  } )
    .Select(tuple => tuple.selectedProfile is not null && !tuple.isPrivacyModeActive);
```

This will need to redefine ShowProfile as `IObservable<bool>` and bind to it in the view using the suffix ^.